src/components/PurposeForUsingExpensifyModal.tsx

```diff
@@ -0,0 +1,178 @@
+        '\n' +
+        "That'll be stored in your My Business Expenses room for your later access. Thanks for asking, and let me know how it goes!",
+    [CONST.INTRO_CHOICES.SUBMIT]:
+        'Hi there, to submit expenses for reimbursement, please:\n' +
+        '\n' +
+        '1. Press the big green + button\n' +
+        '2. Choose Request money\n' +
+        '3. Indicate how much to request, either manually, by scanning a receipt, or by tracking distance\n' +
+        '4. Enter the email address or phone number of your boss\n' +
+        '\n' +
+        "And we'll take it from there to get you paid back. Please give it a shot and let me know how it goes!",
+    [CONST.INTRO_CHOICES.MANAGE_TEAM]:
+        "Great! To manage your team's expenses, create a workspace to keep everything contained:\n" +
+        '\n' +
+        '1. Press your avatar icon\n' +
+        '2. Choose Workspaces\n' +
+        '3. Choose New Workspace\n' +
+        '4. Name your workspace something meaningful (eg, "Galaxy Food Inc.")\n' +
+        '\n' +
+        'Once you have your workspace set up, you can invite your team to it via the Members pane and connect a business bank account to reimburse them!',
+    [CONST.INTRO_CHOICES.CHAT_SPLIT]:
+        'Hi there, to split an expense such as with a friend, please:\n' +
+        '\n' +
+        'Press the big green + button\n' +
+        'Choose *Request money*\n' +
+        'Indicate how much was spent, either manually, by scanning a receipt, or by tracking distance\n' +
+        'Enter the email address or phone number of your friend\n' +
+        'Press *Split* next to their name\n' +
+        'Repeat as many times as you like for each of your friends\n' +
+        'Press *Add to split* when done adding friends\n' +
+        'Press Split to split the bill\n' +
+        '\n' +
+        "This will send a money request to each of your friends for however much they owe you, and we'll take care of getting you paid back. Thanks for asking, and let me know how it goes!",
+};
+
+const menuIcons = {
+    [CONST.INTRO_CHOICES.TRACK]: Expensicons.ReceiptSearch,
+    [CONST.INTRO_CHOICES.SUBMIT]: Expensicons.Scan,
+    [CONST.INTRO_CHOICES.MANAGE_TEAM]: Expensicons.MoneyBag,
+    [CONST.INTRO_CHOICES.CHAT_SPLIT]: Expensicons.Briefcase,
+};
+
+function PurposeForUsingExpensifyModal() {
+    const {translate} = useLocalize();
+    const StyleUtils = useStyleUtils();
+    const styles = useThemeStyles();
+    const {isSmallScreenWidth, windowHeight} = useWindowDimensions();
+    const navigation = useNavigation();
+    const [isModalOpen, setIsModalOpen] = useState(false);
+    const theme = useTheme();
+
+    useEffect(() => {
+        const navigationState = navigation.getState();
+        const routes = navigationState.routes;
+        const currentRoute = routes[navigationState.index];
+        if (currentRoute && NAVIGATORS.CENTRAL_PANE_NAVIGATOR !== currentRoute.name && currentRoute.name !== SCREENS.HOME) {
```

Comment:
Can you please add context behind this logic?

Reply:
If there is a route, we don't want to show the modal unless the page we're on is home or the central pane navigator

Reply:
ok so https://github.com/Expensify/App/pull/32154#pullrequestreview-1837373056 is correct. Just wanted to confirm 🙂

Reply:
Yep that's correct! Thanks for double checking :)